packages/yew-macro/tests/derive_props/pass.rs

```diff
@@ -220,18 +220,6 @@ mod t11 {
     }
 }
 
-mod t12 {
-    #[derive(::std::clone::Clone, ::yew::Properties, ::std::cmp::PartialEq)]
-    pub struct Props<T: ::std::clone::Clone + ::std::cmp::PartialEq> {
-        value: ::std::option::Option<T>,
-    }
-
-    fn optional_prop_generics_should_work() {
-        ::yew::props! { Props::<::std::primitive::bool> { } };
-        ::yew::props! { Props::<::std::primitive::bool> { value: true } };
-    }
-}
-
```

Comment:
Why remove this test? We should not be removing tests. I would like to have this change not come at the cost of functionality. Please add the test back

Reply:
Also I should note that tests being removed all the time is a normal procedure for many developers. Tests can be removed if they do not reflect current reality (i.e. do not pass, in this case because behavior is opposite to what previously was tested for), are covered by other tests (which have been added (will be added per your suggestions) in this PR by the way), or simply poorly written or considered more of a burden for development than the sought for assistance they are supposed to give.

Reply:
> Also I should note that tests being removed all the time is a normal procedure for many developers. Tests can be removed if they do not reflect current reality (i.e. do not pass, in this case because behavior is opposite to what previously was tested for), are covered by other tests (which have been added (will be added per your suggestions) in this PR by the way), or simply poorly written or considered more of a burden for development than the sought for assistance they are supposed to give.

Absolutely, but in this case, the test can be updated to be passed and covers a very important functionality: generic props. We should have a passing test for that feature